changelog/10218.improvement.rst

```diff
@@ -0,0 +1,5 @@
+``@pytest.mark.parametrize()`` (and similar functions) now accepts any ``Sequence[str]`` for the argument names,
```

Comment:
Perhaps we should categorize this as `trivial`? Up to you.

Reply:
It is pretty trivial, but we define `trivial` as "fixing a small typo or internal change that might be noteworthy.", while this *is* user-visible, so I'll keep it as `improvement`.